contracts/drafts/Strings.sol

```diff
@@ -0,0 +1,53 @@
+pragma solidity ^0.5.0;
+
+/**
+ * @title Strings
+ * @dev String operations.
+ * via OraclizeAPI - MIT licence
+ * https://github.com/oraclize/ethereum-api/blob/b42146b063c7d6ee1358846c198246239e9360e8/oraclizeAPI_0.4.25.sol
+ */
+library Strings {
+    /**
+     * @dev Concatenates two strings.
+     */
+    function concatenate(string memory a, string memory b) internal pure returns (string memory concatenatedString) {
+        bytes memory bytesA = bytes(a);
+        bytes memory bytesB = bytes(b);
+        string memory concatenatedAB = new string(bytesA.length + bytesB.length);
+        bytes memory bytesAB = bytes(concatenatedAB);
+        uint concatendatedIndex = 0;
+        uint index = 0;
+        for (index = 0; index < bytesA.length; index++) {
+            bytesAB[concatendatedIndex++] = bytesA[index];
```

Comment:
This can probably be made more efficient by copying a whole word at once if there are more than 32 characters remaining, I'll look into it.